### PR TITLE
Migrate to new RC3 release

### PR DESCRIPTION
**Pull request by wtrocki**
# Motivation

Bump and integrate with RC3 release of the client.
Testing efforts:
- [x] Checking if problem with core instance was resolved (requires openshift config)
- [x] Checking recent changes for Auth on IOS Android `npm run ionic:android`
- [x] Checking subscriptions
- [x] Checking cache overflow issue
- [x] Checking bug fixing done on offline/conflict by @jhellar 

I'm going to cover testing, but also looking for volunteers to test things again.
Once this is done I will merge dev to master.

### CODE REVIEW

**Comment by wtrocki** [comment]
Error:
```
ObservableQuery.js:215 Unhandled GraphQL subscription error Error: GraphQL error: Cannot query field "mutation" on type "TaskSubscription". Did you mean "action"?
```
Working on fixing this.

ping @StephenCoady

**Comment by StephenCoady** [comment]
@wtrocki tested your fix and its working. sorry about that.

**Comment by wtrocki** [comment]
Hey guys. 

I have tried and tested all functionalities in this RC. Things that should be working works. 
Things that had problems still have them :)
Hopefully next RC should be better. This PR has no changes from server as we have made updates in other PR.

I got keycloak running on Android but it failed to redirect. It may be related with the fact that keycloak requires some specific config. We going to still work on Auth as IOS is not working so skipping that part.

**Comment by wtrocki** [comment]
Actually got auth running. Good job @danielpassos @secondsun

**Comment by StephenCoady** [comment]
👀

File: src/app/services/item.service.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export class ItemService {
         if (!subscriptionData.data.tasks.task) {
           return prev;
         }
-        if (subscriptionData.data.tasks.mutation === MutationType.CREATED) {
+        if (subscriptionData.data.tasks.action === MutationType.CREATED) {
```
Comment (ziccardi) [inline src/app/services/item.service.ts]: Wouldn't be better to use a `switch` here?

Comment (wtrocki) [reply src/app/services/item.service.ts]: We have rule that will transform nested if statements if there is more than 3 occurrences. 
Here we have 2 so we are fine with if.

Comment (ziccardi) [reply src/app/services/item.service.ts]: Then I believe we should have at least an `else if`

**Comment by wtrocki** [comment]
@StephenCoady Auth is still in progress so let's not spent more time on it.

**Pull request merged**